### PR TITLE
Add support for Websocket plugin

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**
Update the view for local plugin
Update remix-engine version

### CODE REVIEW

**Comment by LianaHus** [comment]
you have some standard issues

File: src/app/components/local-plugin.js [hunk 1]
```diff
@@ -57,32 +64,41 @@ module.exports = class LocalPlugin {
...
+    //     </div>`
+    //   : yo`<div class="radio">
+    //       <label for="${label}">
+    //         <input type="radio" name="location" onclick="${e => this.updateLoc(e)}" value="${label}" id="${label}" />${displayN}</label>
+    //     </div>`
+    //   return yo`<div>
+    //     ${radioButton}
+    //   </div>`
+    // }
+
```
Comment (LianaHus) [inline src/app/components/local-plugin.js]: remove?

File: src/app/components/local-plugin.js [hunk 2]
```diff
@@ -97,11 +99,17 @@ module.exports = class LocalPlugin {
         <label for="plugin-url">Url <small>(required)</small></label>
         <input class="form-control" onchange="${e => this.updateUrl(e)}" value="${url}" id="plugin-url" placeholder="ex: https://localhost:8000">
       </div>
-      <div class="form-group">
+      <h6>Type of connection <small>(required)</small></h6>
+      <div class="form-check form-group">
+        ${radioSelection('type', 'iframe', 'Iframe')}
```
Comment (LianaHus) [inline src/app/components/local-plugin.js]: could you change it to "IFrame"?

Comment (GrandSchtroumpf) [reply src/app/components/local-plugin.js]: It's supposed to be camelCase.

**Pull request merged**